tableau-databricks/manifest.xml

```diff
@@ -142,10 +142,11 @@ limitations under the License.
         <!-- ### Properties set to control the metadata queries -->
         <customization name='CAP_FAST_METADATA' value='no' />
         <customization name='CAP_ODBC_SUPPRESS_PREPARED_QUERY_FOR_NON_COMMAND_QUERIES' value='yes' />
-        <customization name='CAP_ODBC_METADATA_SUPPRESS_PREPARED_QUERY' value='no' />
-        <customization name='CAP_ODBC_METADATA_SUPPRESS_EXECUTED_QUERY' value='no' />
-        <customization name='CAP_ODBC_METADATA_SUPPRESS_SQLCOLUMNS_API' value='yes' />
-        <customization name='CAP_ODBC_METADATA_SUPPRESS_SELECT_STAR' value='no' />
+        <customization name='CAP_ODBC_METADATA_SUPPRESS_PREPARED_QUERY' value='yes' />
+        <customization name='CAP_ODBC_METADATA_SUPPRESS_EXECUTED_QUERY' value='yes' />
+        <customization name='CAP_ODBC_METADATA_SUPPRESS_SQLCOLUMNS_API' value='no' />
+        <customization name='CAP_ODBC_METADATA_SUPPRESS_SELECT_STAR' value='yes' />
+        <customization name='CAP_ODBC_SUPPRESS_CATALOG_NAME' value='yes'/>
```

Comment:
If we suppress the catalog name, we may actually break the managed catalog because we're dropping the catalog hierarchy. @CJStuart does the current SDK support displaying the `catalog.schema.table` hierarchy similarly to PBI? If so, we should try to expose it in the catalog. cc @alismess-db